web/Web/web/templates/base.html

```diff
@@ -68,7 +68,7 @@
           <a class="nav-item nav-link {% if 'CAL' in request.resolver_match.app_names %}text-primary{% endif %}" href="{% url 'CAL:main' %}">Discovery</a>
           {% endif %}
 
-          <a class="nav-item nav-link {% if 'Review' in request.resolver_match.app_names %}text-primary{% endif %}" href="#">Review</a>
+          <a class="nav-item nav-link {% if 'Review' in request.resolver_match.app_names %}text-primary{% endif %}" href="{% url 'review:main' %}">Review</a>
```

Comment:
it should be lower case `review` in `{% if 'Review' in request ...`

Reply:
fixed